tests/subsys/fs/fat_fs_api/src/test_fat.h

```diff
@@ -1,13 +1,15 @@
 /*
  * Copyright (c) 2016 Intel Corporation.
  * Copyright (c) 2020 Nordic Semiconductor ASA
+ * Copyright (c) 2022 Husqvarna AB
```

Comment:
It is 23 now, took us long to review.

Reply:
Shall I also update the git commit date?